### PR TITLE
Functional checklist

### PR DESCRIPTION
**Pull request by mariem-abid**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 8.x
| Description?  | Improve the **validation process** of the modules and themes on the **disneystore**.  The requests of "Sarah-Jane Le Corre" (Head of the care team)
| Fixed ticket? | no

### CODE REVIEW

File: module-theme-functional-checklist/_index.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+* Verification of the messages for updating and deleting.
+* Text messages on errors in case of incorrect input.
+
+###Inputs:
+All input types must be checked before saving them. Be sure that:
+* You can only put numbers in a numeric field.
+* Email input must accept only email format.
+* The real number must include both rational and irrational numbers.
+
+###Social Networks:
```
Comment (kpodemski) [inline module-theme-functional-checklist/_index.md:40]: @mariem-abid this definitely doesn't apply to any best practices, etc. It's Addons Marketplace specific thing

**Review by kpodemski — COMMENTED** [review]
I think we might have a section helping module developers making sure their products work well, but Addons Marketplace is not an only markeplatce in PrestaShop ecosystem. This text shouldn't have any specifics required by it.

**Review by matks — CHANGES_REQUESTED** [review]
Hello @mariem-abid I think indeed there is a misunderstanding ^^ this repository is not where such document should go.

I think it should go on www.prestashop.com or https://addons.prestashop.com/

**Review by matks — CHANGES_REQUESTED** [review]
Blocked while waiting for Steering Committee decision

https://github.com/PrestaShop/docs/pull/1369#issuecomment-1114640784

**Comment by matks** [comment]
I close this as the linked contribution was closed https://github.com/PrestaShop/docs/pull/1369#issuecomment-1114640784

**Pull request closed without merging**